grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -180,6 +180,11 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
             return hasPrepaymentInvoice && !hasRegularInvoice && order.placed && !invoicedFullAmount
         }
 
+        // check if by any chance it was invoiced as cancelled
+        if (totalAdjustments == 0) {
+            return order.placed && !hasRegularInvoice
```

Comment:
is there a case where we do have a regular invoice but it doesn't invoice for the zero adjustment? Wouldn't in that case the adjustment still be invoiceable?

Reply:
I don't think there is such a case.
I was not aware of the case for invoicing adjustments with amount 0 and this check should handle it.

So in my understanding, if an adjustment already has a regular incoice it was probably already invoiced in some way and since totalAdjusmten amount specified is zero we can assume it was invoiced as 0